src/wallet/rpcwallet.cpp

```diff
@@ -841,11 +841,16 @@ UniValue listaddressgroupings(const UniValue& params, bool fHelp)
     if (!EnsureWalletIsAvailable(fHelp))
         return NullUniValue;
 
-    if (fHelp)
+    if (fHelp || params.size() > 1)
         throw runtime_error(
-            "listaddressgroupings\n"
+            "listaddressgroupings ( asOfHeight )\n"
             "\nLists groups of transparent addresses which have had their common ownership\n"
             "made public by common use as inputs or as the resulting change in past transactions.\n"
+            "\nArguments:\n"
+            "1. asOfHeight       (numeric, optional) Execute the query as if it were run when\n"
+            "                    the blockchain was at the height specified by this argument.\n"
+            "                    The default is to use the entire blockchain that the node is\n"
+            "                    aware of.\n"
```

Comment:
Document what happens if `asOfHeight` is greater than the height of the last known block.